ando/tools/generator/AnDOGenerator.py

```diff
@@ -216,14 +228,27 @@ def validate(self):
         """
         Validate the generated structure using the AnDO validator
 
-        Returns:
-            (bool): True if validation was successful. False if it failed.
+        Returns
+        ----------
+        bool
+            True if validation was successful. False if it failed.
         """
 
         raise NotImplementedError('Ando validation is not implemented yet.')
 
 
 def create_file(source, destination, mode):
+    """
+
+    Parameters
+    ----------
+    source:
+    destination:
+    mode:
+
+    Returns
+    -------
+    """
```

Comment:
Is this the correct way of documenting 
```suggestion
    """
    Create a file at a destination location

    Parameters
    ----------
    source: str
        Source location of the file.
    destination: str
        Destination location of the file.
    mode: str
        File creation mode. Valid parameters are 'copy', 'link' and 'move'.

    Returns
    -------
    """
```